pike/test/lock.py

```diff
@@ -136,7 +137,7 @@ def test_deny_write(self):
                             disposition=pike.smb2.FILE_OPEN,
                             options=pike.smb2.FILE_DELETE_ON_CLOSE).result()
 
-        cases = ((offset,size) for offset in xrange(0, 16) for size in xrange(0, 16))
+        cases = ((offset,size) for offset in range(0, 16) for size in range(0, 16))
```

Comment:
it's sending 256 write requests for different ranges, it probably could be optimized;

i'm trying to avoid making too much unnecessary churn at this point though